prisma/schema.prisma

```diff
@@ -32,12 +32,41 @@ model Team {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
+  competitionTeams CompetitionTeams[]
+
+  @@map("competitions")
+}
+
+model Team {
+  id Int @id @default(autoincrement())
+  title String
+  key String @unique
+  primaryColor String
+  secondaryColor String
+  tertiaryColor String
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  competitionTeams CompetitionTeams[]
   teamMembers TeamMember[]
   teamSheets TeamSheet[]
 
   @@map("teams")
 }
 
+model CompetitionTeams {
+  competition Competition @relation(fields: [competitionId], references: [id])
+  competitionId Int
+  team Team @relation(fields: [teamId], references: [id])
+  teamId Int
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  @@id([competitionId, teamId])
```

Comment:
i think we should just put in the `id` as normal instead of this, if we ever build that admin section we will want to be able to delete by id